website/docs/error-reference.md

```diff
@@ -702,6 +702,68 @@ you'll get this error—the latter export is redundant, as it's implied by expor
 
 To fix this error, simply remove the more specific export.
 
+## 3717
+
+> This error is specific to Stripe's custom `--stripe-packages` definition. If
+> you are at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+This error means that you're using a name that's private in another package, and
+not epxorted.
+
+To fix this error, you'll need to either export the symbol in the other package,
+or figure out a way around using it.
+
+## 3718
+
+> This error is specific to Stripe's custom `--stripe-packages` definition. If
+> you are at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+This error means that you're using a symbol from a packge that's not imported.
```

Comment:
I've tried to avoid using the word "symbol" to refer to definitions in error messages. I prefer using "definition" or "constant" instead.

Reply:
That sounds good, I'll take a pass over the docs to fix that.

Reply:
(i fixed this in my edits)